app/serializers/user_serializer.rb

```diff
@@ -3,7 +3,7 @@ class UserSerializer
   include RecentLinkSerializer
   include MediaLinksSerializer
 
-  attributes :id, :display_name, :credited_name, :email, :created_at,
+  attributes :id, :login, :display_name, :credited_name, :email, :created_at,
     :updated_at, :type, :firebase_auth_token, :global_email_communication,
     :project_email_communication, :beta_email_communication,
     :slug, :max_subjects, :uploaded_subjects_count, :admin
```

Comment:
> Yes it can be. 

Is in response to?

> identity? will never be true on creation since that query can't run on the unpersisted records.

(I can't see how that's possible)

or

> Since User#display_name can be unrestricted, shouldn't UserGroup#display_name be unrestricted?

If that's the case, I'll remove the display_name validation, but validate the name attribute with the same format as User#login?